blog/2020-10-08-FacilityLocationProblemQuickstart.adoc

```diff
@@ -0,0 +1,98 @@
+= New quickstart for the facility location problem
+:imagesdir: 2020/10/08
+:page-interpolate: true
+:awestruct-author: yurloc
+:awestruct-layout: blogPostBase
+:awestruct-tags: [use case, facility location problem, quarkus]
+
+:awestruct-share_image_filename: CreditDisputeProcessDiagram.png
```

Comment:
No empty line above + I think this filename is wrong

Reply:
Good catch, it's a copy-paste residue. What does this attribute do? Can I delete it? I'm not sure which image to use here.

Reply:
Re-use the most flashy of your images in the blog below here :)
The image doesn't show up in the blog (so no issue to reuse it), but it shows up when this URL is linked on twitter or facebook, along with the title. Think something that people would click on. For this kind of blog, an solved or unsolved screenshot is ideal.